browser-test/src/support/index.ts

```diff
@@ -8,11 +9,13 @@ export { AdminTIGroups } from './admin_ti_groups'
 export { ApplicantQuestions } from './applicant_questions'
 export { clickAndWaitForModal, waitForPageJsLoad } from './wait'
 
-const {
-  BASE_URL = 'http://civiform:9000',
-  TEST_USER_LOGIN = '',
-  TEST_USER_PASSWORD = '',
-} = process.env
+import {
```

Comment:
just curious: why this change?

Reply:
so that BASE_URL could be use by other files in `src/support` without creating a circular dependency